libecho/src/RawReceiver.h

```diff
@@ -0,0 +1,22 @@
+#include <QtMultimedia/QAudioFormat>
+#include <memory>
+#include "AudioInput.h"
+#include "IReceiver.h"
+
+class RawReceiver : public IReceiver {
+public:
+	RawReceiver(int loFreq, int hiFreq, double magLimitBegin, double magLimitEnd, int winSize);
```

Comment:
Maybe add default values for some arguments ^^

Reply:
I disagree. It doesn't make using this module easier and may cause little, but annoying bugs. Also, these parameters don't have an obvious default.